notebooks/data_processing.ipynb

```diff
@@ -0,0 +1,795 @@
+{
```

Comment:
<p>Xarray is one of the best auto-explainatory data structures because it immediately tells us what are the dependent variables and what are the independent ones. Here, elevation only depends on <code>location_id</code> so we should only have <code>location_id</code> in the first entry of the tuple.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/patel-zeel/kdd24/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>